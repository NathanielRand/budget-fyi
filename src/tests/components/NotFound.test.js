import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from "../../components/NotFound";
import { getExpectedBodyHash } from "twilio/lib/webhooks/webhooks";

test("Should render page not found", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
