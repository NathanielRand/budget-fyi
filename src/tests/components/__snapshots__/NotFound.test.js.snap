// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render page not found 1`] = `
<div>
  <p>
    Whomp! Try a different page my friend...or go home below
  </p>
  <Link
    to="/"
  >
    Phone Home Here
  </Link>
</div>
`;
